### DISCUSSION TITLE
Possible Killer Bunny implamentation?

### DISCUSSION THREAD

**Discussion by Daralexen05** [Faboslav/friends-and-foes#218] [Ideas]
One of the other points of this mod is to give forgotten mobs that were left exclusive to /Summon Commands or Creative such as the Illusionist and Zombie Horse second chances at life in a very believable and vanilla-friendly way. I know that the Indev Giant has been denied countless times from being in this mod due to not fitting modern Minecraft due to its sheer size making navigating more complex terrain difficult and its stupidly OP stats. However, what about another /summon exclusive in the form of the Killer Bunny? It was a rare hostile variant of Rabbits that spawned in 1/1000 Rabbits. It dealt massive damage similar to Indev Giants and they attacked all Players, Wolves, and Foxes. but they were glass cannons and still had the pitifully low HP of their standard passive counterparts. Maybe this mod can give the Killer Bunny a second chance at being able to naturally spawn. Maybe when a Rabbit has been in the Nether for too long, is struck by lightning, or maybe a Rabbit trap during Thunderstorms similar to Monster Horses.

**Comment by Draken-arch** [comment]
I agree with this!
